app/webpacker/styles/thisapp/overrides.scss

```diff
@@ -0,0 +1,16 @@
+.govuk-form-group.inline {
+  .govuk-form-group {
+    display: inline-block;
+  }
+}
+
+td {
+  .govuk-form-group {
+    margin-bottom: 0;
+  }
+}
+
```

Comment:
These overrides are dangerous.

Better to use the helper classes in the HTML as they were intended:

`govuk-!-margin-bottom-0` and `govuk-!-display-inline-block`

Reply:
Good call - I didn't know about those classes, now I do  :)

I've taken out the margin-bottom override, but I can't get a custom class name to pass through the `govuk_form_builder (...) govuk_collection_select` helper to actually appear in the HTML.  (`html_options: {class: '...'}` doesn't seem to work). 

OK to stick with the `govuk-form-group.inline` hack for now?

Reply:
as discussed elsewhere, I've kept the CSS hack but with a more specific name (`.display-child-forms-inline`) and an explanatory comment